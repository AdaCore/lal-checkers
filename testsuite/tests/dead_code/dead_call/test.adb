procedure Test is
   procedure P (X : out Integer) with Import;

   X : Integer;
begin
   if False then
      P (X);
   end if;
end A;
