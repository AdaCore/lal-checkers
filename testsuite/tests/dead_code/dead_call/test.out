{
  "always": [
    "assign1"
  ], 
  "le_t_eq_v": [
    "assign1"
  ]
}
